packages/create-llama/templates/python.ts

```diff
@@ -21,6 +21,29 @@ const getAdditionalDependencies = (vectorDb?: TemplateVectorDB) => {
       });
       break;
     }
+    case "pg": {
+      dependencies.push(
+        ...[
+          {
+            name: "psycopg2-binary",
+            version: "^2.9.1",
+          },
+          {
+            name: "sqlalchemy",
+            version: "^2.0.25",
```

Comment:
seems like that's already in `llama-index`: https://github.com/run-llama/llama_index/blob/7ab3f5008696e99ee78d06312b124d785d8c9fd8/pyproject.toml#L48C7-L48C7

also, the other dependencies are optional in `llama-index` - is there an easy way to use the version number specified in `llama-index`? e.g. by not specifying a number here?

Reply:
Yes, we can add the `postgres` extras attribute to the `llama-index` dependency in pyproject.toml but the python render codebase does not support that, let me work further on that.